### PR TITLE
fix(hooks): use the right data to remove old repository event

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

File: engine/hooks/scheduler_repository_event_retry.go [hunk 1]
```diff
@@ -49,18 +49,18 @@ func (s *Service) manageOldRepositoryEvent(ctx context.Context) {
...
-	find, err := s.Cache.Get(repoEventKey, &repoEventTmp)
+	find, err := s.Cache.Get(repoEventUUID, &repoEventTmp)
 	if err != nil {
 		return err
 	}
 	if !find {
-		log.Info(ctx, "repository event %s does not exist anymore.", repoEventTmp.GetFullName())
-		if err := s.Dao.RemoveRepositoryEventFromInProgressList(ctx, repoEventTmp); err != nil {
+		log.Info(ctx, "repository event %s does not exist anymore.", repoEventUUID)
+		if err := s.Dao.RemoveRepositoryEventFromInProgressList(ctx, repoEventUUID); err != nil {
```
Comment (sguiheux) [inline engine/hooks/scheduler_repository_event_retry.go:63]: the problem was here

File: engine/hooks/scheduler_repository_event_retry.go [hunk 2]
```diff
@@ -80,13 +80,13 @@ func (s *Service) checkInProgressEvent(ctx context.Context, repoEventKey string)
...
-	find, err = s.Cache.Get(repoEventKey, &hre)
+	find, err = s.Cache.Get(repoEventUUID, &hre)
 	if err != nil {
 		return sdk.WrapError(err, "unable to retrieve repository event")
 	}
 	if !find {
-		log.Info(ctx, "repository event %s does not exist anymore.", hre.GetFullName())
-		if err := s.Dao.RemoveRepositoryEventFromInProgressList(ctx, hre); err != nil {
+		log.Info(ctx, "repository event %s %s does not exist anymore.", repoEventUUID, repoEventTmp.GetFullName())
+		if err := s.Dao.RemoveRepositoryEventFromInProgressList(ctx, repoEventUUID); err != nil {
```
Comment (sguiheux) [inline engine/hooks/scheduler_repository_event_retry.go:89]: The problem was here

**Review by yesnault — APPROVED** [review]

**Review by fsamin — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-cds#21580.0 ✔ 
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#21580.0 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Pull request merged**